misc/pp_mixed_scenarios/README.md

```diff
@@ -0,0 +1,65 @@
+# Post-Processing with Mixed Scenarios
+
+This script is used for post-processing results generated from mixed scenarios. It is essential that
+the directory structure is properly set up for the script to function correctly. Please follow the
+guidelines outlined below.
+
+---
+
+## Directory Structure
+
+The working directory should contain the following folders and files:
+---
+
+## File Details
+
+### User-Provided Files:
+
+1. **`mixed_scenarios_desc.yaml`**
+    - This file contains the configuration for processing different scenarios. Its structure should
+      follow the example provided below:
+        ```yaml
+           disease: lf                       # Valid are `lf`, `oncho` and `trachoma`
+           default_scenario: scenario_0      # Default scenario to process results from
+           overridden_ius:
+               scenario_1: [CAF09661, CAF09662]  # IUs to take from scenario_1
+               scenario_2: [CAF09663]            # IUs to take from scenario_2
+           scenario_name: scenario_x1        # Name applied to the processed scenario set
+        ```
+
+
+2. **`input`**
+    - Must include the following:
+        - **`canonical_results`**: Directory containing subfolders for each scenario, such as
+          `scenario_0`, `scenario_1`, etc.
+        - **`PopulationMetadatafile.csv`**: A CSV file with population metadata required for further
+          processing.
+
+### Created By Script:
+
+1. **`output`**
+    - This directory will be created automatically by the script. It contains the following:
+        - **`canonical_results`**: Subfolders for processed scenarios, which are created based on
+          the configuration in `mixed_scenarios_desc.yaml`.
+        - **`mixed_scenarios_metadata.json`**: A JSON metadata file generated during the processing.
```

Comment:
❓ And presumably the aggregate files? I think it would be good to specifically point where the Africa level aggregate as well as the iu level aggregate files are